docs/specs/src/diagram-1.0.0/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,154 @@
+name: CI
+
+on:
+  # Run on all pull requests that change code.
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - LICENSE
+      - .editorconfig
+  # Run every time a code change is pushed.
+  push:
+    paths-ignore:
+      - 'README.md'
+      - LICENSE
+      - .editorconfig
+  # Test if things still work each Tuesday morning.
+  # This way we will catch incompatible pandoc changes in a timely
+  # manner.
+  schedule:
+    # At 4:27am each Tuesday
+    - cron: '27 4 * * 2'
+
+jobs:
+  Asymptote:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        pandoc:
+          - latest
+
+    container:
+      image: pandoc/latex:${{ matrix.pandoc }}-ubuntu
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          apt-get -q --no-allow-insecure-repositories update && \
+          apt-get install --no-install-recommends --assume-yes \
+            make inkscape asymptote
+
+      - name: Test
+        run: 'make test-asymptote'
+
+  GraphViz:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        pandoc:
+          - latest
```

Comment:
is a matrix strategy necessary if only the `latest` version is getting used?

Reply:
this is just a bunch of junk that came with a Pandoc extension I'm using to render diagrams -- thought I'd put it in here so people don't have to go searching for it. I'm guessing it'll never get used; I didn't even know it was there :)